packages/calcite-components/src/demos/combobox.html

```diff
@@ -1164,6 +1164,47 @@ <h3 class="child right-aligned-text">Validation message</h3>
           </div>
         </div>
       </div>
+
+      <div class="parent">
+        <h3 class="child right-aligned-text">Theme</h3>
```

Comment:
I might be wrong, but I think since we started adding custom theme stories, we don't add tokens to demo anymore.

Reply:
Interesting, wasn't aware of this. Prefer to keep it for now and we can remove all the theme related demos if thats the way forward.